### PR TITLE
docs: add links to the issue tracker, to developer/issues

### PR DESCRIPTION
**Pull request by ig3**
#### Description of Change
This change adds links to the `electron/electron` issue tracker on the development/issues page, to make it easier to get to the issues.

#### Checklist

- [ ] PR description included and stakeholders cc'd
- [x] relevant documentation is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)

I don't know who the stakeholders are or how to cc them, sorry.

#### Release Notes

Notes: 'no-notes'

### CODE REVIEW

**Comment by ig3** [comment]
I don't understand "you should update your pull request title to start with a semantic prefix." in the comment from the bot above. The title began with "Docs: " and I changed it to "docs: " in case lower case is required. But the requirement to change the title remains. Let me know how I can satisfy the bot and I'm happy to change the title again.

**Comment by zcbenz** [comment]
This seems to be basic GitHub usage information, /cc @electron/wg-ecosystem for whether we should it in the development documentations.

File: docs/development/issues.md [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
 * [How to Contribute to Issues](#how-to-contribute-to-issues)
 * [Asking for General Help](#asking-for-general-help)
 * [Submitting a Bug Report](#submitting-a-bug-report)
 * [Triaging a Bug Report](#triaging-a-bug-report)
 * [Resolving a Bug Report](#resolving-a-bug-report)
 
+## Where to Find and Submit Issues
+
+Issues for `electron/electron` are in the GitHub
+[issue tracker](https://github.com/electron/electronjs.org/issues/).
```
Comment (malept) [inline docs/development/issues.md]:
```suggestion
[issue tracker](https://github.com/electron/electron/issues).
```

File: docs/development/issues.md [hunk 2]
```diff
@@ -1,11 +1,18 @@
 # Issues In Electron
 
+* [Where to Find and Submit Issues](#where-to-find-and-submit-issues)
 * [How to Contribute to Issues](#how-to-contribute-to-issues)
 * [Asking for General Help](#asking-for-general-help)
 * [Submitting a Bug Report](#submitting-a-bug-report)
 * [Triaging a Bug Report](#triaging-a-bug-report)
 * [Resolving a Bug Report](#resolving-a-bug-report)
 
+## Where to Find and Submit Issues
```
Comment (malept) [inline docs/development/issues.md]: I don't think there needs to be a new section for this, just add a link to the first instance of:

> new issue in the `electron/electron` issue tracker.

File: docs/development/issues.md [hunk 3]
```diff
@@ -30,6 +37,9 @@ contributing, and more. Please use the issue tracker for bugs only!
 
 ## Submitting a Bug Report
 
+To submit a bug report:
+[open an issue](https://github.com/electron/electronjs.org/issues/new).
+
 When opening a new issue in the `electron/electron` issue tracker, users
```
Comment (malept) [inline docs/development/issues.md]:
```suggestion
When opening a new issue in the [`electron/electron` issue tracker](https://github.com/electron/electron/issues/new/choose), users
```

File: docs/development/issues.md [hunk 4]
```diff
@@ -30,6 +37,9 @@ contributing, and more. Please use the issue tracker for bugs only!
 
 ## Submitting a Bug Report
 
+To submit a bug report:
+[open an issue](https://github.com/electron/electronjs.org/issues/new).
```
Comment (malept) [inline docs/development/issues.md]: I don't think there needs to be a new paragraph for this.

File: docs/development/issues.md [hunk 5]
```diff
@@ -1,11 +1,18 @@
 # Issues In Electron
 
+* [Where to Find and Submit Issues](#where-to-find-and-submit-issues)
```
Comment (malept) [inline docs/development/issues.md]:
```suggestion
```

File: docs/development/issues.md (hunk 1 above)
Comment (ig3) [reply docs/development/issues.md]: PR updated as recommended.

File: docs/development/issues.md (hunk 2 above)
Comment (ig3) [reply docs/development/issues.md]: PR updated as recommeded.

File: docs/development/issues.md (hunk 3 above)
Comment (ig3) [reply docs/development/issues.md]: PR updated as recommended

File: docs/development/issues.md (hunk 4 above)
Comment (ig3) [reply docs/development/issues.md]: PR updated as recommended

File: docs/development/issues.md (hunk 5 above)
Comment (ig3) [reply docs/development/issues.md]: PR updated as recommended

**Comment by ig3** [comment]
@erickzhao concurrent with you reviewing and approving, I was making the changes recommended by @malept. I trust this makes it even better ;-)

**Pull request merged**